### PR TITLE
Caching local variables, reformating code, processing null-var cases

### PR DESCRIPTION
**Pull request by DeaThunder**

### CODE REVIEW

File: InteractToggleGameObject.cs [hunk 1]
```diff
@@ -9,6 +9,14 @@ public class ToggleObject : UdonSharpBehaviour
...
     public GameObject toggledObject;
     
+    private void Start()
+    {
+        if (toggledObject == null)
+        {
+            toggledObject = this.gameObject;
+        }
+    }
+
```
Comment (Zyoh) [inline InteractToggleGameObject.cs]: Toggling itself wouldn't really work - no way to enable it again once it's off.

File: JumpPads.cs [hunk 2]
```diff
@@ -9,33 +9,47 @@ public class JumpPads : UdonSharpBehaviour
...
-            if (inProx) ExitProxAction();
             inProx = false;
+            ExitProxAction();
         }
     }
 
-    public void ProximityAction()
+    private void ProximityAction()
     {
-        jumpHeight = Networking.LocalPlayer.GetJumpImpulse();
```
Comment (Zyoh) [inline JumpPads.cs:33]: Keep this check to revert to jump impulse directly before jumping instead of when script was loaded.

**Review by Zyoh — CHANGES_REQUESTED** [review]

File: JumpPads.cs (hunk 2 above)
Comment (DeaThunder) [reply JumpPads.cs:33]: Reverted

File: InteractToggleGameObject.cs (hunk 1 above)
Comment (DeaThunder) [reply InteractToggleGameObject.cs]: Removed

**Review by Zyoh — APPROVED** [review]

**Pull request merged**